framework/components/ecl-files/_files.scss

```diff
@@ -1,289 +1,193 @@
-
-      color: $ecl-color-primary;
-      margin-left: 0.525em !important;
-      position: relative;
-      transition: all 0.3s ease 0s;
-    }
-
-    &.is-collapsed {
-      &::after {
-        transform: rotate(180deg);
-      }
-    }
-  }
-
-  .file__translations-list {
-    list-style: none;
-    margin: 0.944em 0 0;
-    padding: 0;
-    text-align: left;
-
-    .btn--condensed {
-      margin-right: -1em;
-    }
-
-    // Make it self scaling.
-    @media (max-width: map-get($ecl-grid-breakpoints, 'sm')) {
-      .btn--condensed {
-        margin-left: -1em;
-        margin-right: 0;
-      }
-    }
-  }
+.ecl-file__translations-toggle {
+  font-weight: normal;
+}
 
-  .file__translations-item {
-    border-top: 1px solid map-get($ecl-colors, 'grey-50');
+.ecl-file__translations-toggle[aria-expanded='false'] {
+  &::after {
+    @extend %ecl-icon--after;
 
-    &:last-child {
-      border-bottom: 1px solid map-get($ecl-colors, 'grey-50');
-    }
+    @include ecl-icon('down');
   }
+}
 
-  .file__metadata {
-    @extend .col-md-6 !optional;
-    @extend .col-sm-6 !optional;
-    @extend .col-xs-12 !optional;
+.ecl-file__translations-toggle[aria-expanded='true'] {
+  &::after {
+    @extend %ecl-icon--after;
 
-    padding-left: 0;
-    padding-right: 0;
+    @include ecl-icon('up');
   }
+}
 
-  // "Looking for another language ...".
-  .file__translations-details {
-    @extend .file__translations-item;
+.ecl-file__translations-list {
+  list-style: none;
+  margin-bottom: 0;
+  margin-top: 0;
+  text-align: left;
+}
 
-    padding-top: 0.5em;
-    font-size: map-get($ecl-font-size, 's');
+.ecl-file__translations-item {
+  border-top: 1px solid map-get($ecl-colors, 'grey-50');
+  display: flex;
+  flex-direction: column;
+  padding-bottom: map-get($ecl-spacing, 'xxs');
+  padding-top: map-get($ecl-spacing, 'xxs');
+
+  &:last-child {
+    border-bottom: 1px solid map-get($ecl-colors, 'grey-50');
   }
 }
 
-/*
-File link
-*/
-.file--link {
-  background-color: transparent;
-
-  .file__metadata--link {
-    @extend .col-md-10 !optional;
-    @extend .col-sm-10 !optional;
-    @extend .col-xs-10 !optional;
-
-    margin-bottom: 0;
-    padding-left: 0;
-    padding-right: 0;
-    line-height: 1;
-  }
+.ecl-file__translations-metadata {
+  flex-grow: 1;
+}
 
-  .ecl-icon--file-link {
-    background: url("../images/svg/icons/link.svg") no-repeat center 10%;
-    width: 2.8em;
-  }
+.ecl-file__translations-title {
+  font-size: map-get($ecl-font-size, 's');
+  font-weight: bold;
 }
 
-.no-svg {
-  .ecl-icon--file-link {
-    background-image: url("../images/png/icons/link.png");
-  }
+.ecl-file__translations-info {
+  font-size: map-get($ecl-font-size, 'xs');
+  margin-top: map-get($ecl-spacing, 'xxs');
 }
 
-@include ecl-media-breakpoint-up(sm) {
-  .file__icon {
-    width: auto;
-    padding: 0;
-  }
+.ecl-file__translations-download {
+  align-self: flex-start;
+  font-weight: normal;
+  margin-left: -1rem;
```

Comment:
That's because it is a button without borders (modifier "secondary"), which comes with a padding. So to align the text I had to add some negative margin